_includes/footer.html

```diff
@@ -46,7 +46,7 @@ <h4>Support our work</h4>
   <div class="footer-copyright">
     <div class="container">
       <p>Humanitarian OpenStreetMap Team is a 501(c)(3) not-for-profit organization and global community. Learn more about <a href="https://www.openstreetmap.org/about">OpenStreetMap</a>.</p>
-      <img src="{{ "/images/hot-logo-icon-white.svg" | prepend: site.baseurl }}">
+      <a href="https://www.hotosm.org/"><img src="{{ "/images/hot-logo-icon-white.svg" | prepend: site.baseurl }}"></a>
```

Comment:
Instead of `href="https://www.hotosm.org/"` just `href="/"` should work. That's the style we follow for [header nav](https://github.com/hotosm/hotosm-website/blob/gh-pages/_includes/nav.html#L12-L15) as well.